### PR TITLE
Added status codes back instead of revert, added proposed Id's

### PR DESCRIPTION
**Pull request by brendaloya**
-Added the status code back to avoid reverting. 
-Updated the interface
-Added proposed Id's
-Added proposed status codes

### CODE REVIEW

**Comment by brendaloya** [comment]
@aesedepece  take a look at these updates

File: README.md [hunk 1]
```diff
@@ -63,3 +77,14 @@ Latest value available for the requested `id`.
 
 To accommodate for decimal values (which are common in prices and rates) this is expected to be multiplied by a power of ten such as `10**9` or `10**18`.
 The specific power of ten MUST explicitly be stated by the `id` used as input.
+
+##### `int status`
+
+Describes the value's status. For consistency the statsus codes will corespond to the codes used for [HTTP](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) status codes
```
Comment (aesedepece) [inline README.md:83]:
```suggestion
Describes the value's status. For consistency the status codes will correspond to the codes used for [HTTP](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) status codes.
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,21 @@ It is up to the implementor of an EIP-2362 compliant contract to decide which `i
 
 These `id`s MUST however be uniform across all providers: multiple EIP-2362 compliant contracts can give different values and timestamps for the same `id`, but they must be referring to the same data point.
 	
-**TODO:** The specifics of how to generate or agree on the `id` of a data point have not been specified yet.
+Proposed Id generation fields: 
+A complet list is kept on Google doc [here](https://docs.google.com/spreadsheets/d/15TPUuPWrxh7eMGaJKIupOh6l5yG-abn8UJejVhHCWV8/edit?usp=sharing).
```
Comment (aesedepece) [inline README.md:49]:
```suggestion
A complete list is kept on Google doc [here](https://docs.google.com/spreadsheets/d/15TPUuPWrxh7eMGaJKIupOh6l5yG-abn8UJejVhHCWV8).
```

File: README.md [hunk 3]
```diff
@@ -27,11 +27,11 @@ The pull-based interface specs:
 
 ```solidity
 interface Eip2362Oracle {
-	function valueFor(bytes32 id) external view returns (uint timestamp, int value);
+	function valueFor(bytes32 id) external view returns (uint timestamp, int value, int status);
 }
 ```
 
-- `valueFor` MUST revert if the value for an `id` is not available yet.
+- `valueFor` MUST return a status code of 404 if the value for an `id` is not available yet.
```
Comment (Amxx) [inline README.md:34]: Why 404. There will be a confusion between http code 404 and this. They are definitelly two very different things !

Also, why `int` ?

File: README.md (hunk 1 above)
Comment (Amxx) [reply README.md:83]: I personally believe that reusing HTTP codes here will cause more harm than good...
We do not control HTTP status code, and there might be some confusion. If not now, then in the future.

In particular, in a client/server context, 4xx are client errors, and 5xx are server errors. If my smart contract (client) calls an EIP2362 compliant oracle, is the oracle the server? is the API behind the oracle the server? what about oracles hierarchies?
When a result is not (yet) available, should it be 404? 405? 500? 502? 503? Depending on the situation (unknown ID, unavailable parent oracle, ...), they might all make some sens...

Also, If we really want to go with HTTP status code, why use an `int`? HTTP status code are uint between 100 and 599, so a `uint16` is more than enough.

Comment (brendaloya) [reply README.md:83]: Hi Hadrien @Amxx ! I think we are all open to creating our own ADO codes or using something else, this is just one of the ideas that I could recall from one of the calls. Would you like to propose an alternative?

File: README.md (hunk 2 above)
Comment (brendaloya) [reply README.md:49]: @aesedepece  and @Amxx , Also, this list is also a proposal, feel free to add or edit. thanks!

**Review by themandalore — APPROVED** [review]

**Pull request merged**